app/views/components/fields/field_accessible_location_autocomplete_with_header.scala.html

```diff
@@ -0,0 +1,83 @@
+@*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import services.view.AutoCompleteItem
+@import views.helpers.CountryAutoCompleteHelper
+@import views.html.components.inputs.input_accessible_location_autocomplete
+@import views.html.helper._
+@(field: Field,
+  label: String,
+  labelClass: Option[String],
+  section: Option[String] = None,
+  bodyHtml: Option[Html] = None,
+  hintParagraphs: List[String] = List.empty,
+  emptySelectValue: String,
+  items: List[AutoCompleteItem],
+  isPageHeading: Boolean,
```

Comment:
Shouldn't we remove pageHeading from components?

Reply:
Should we? This is used on those pages that only have a single form country element with no body text (so just heading then form field component) such as country of routing.

Reply:
I have seen amendments by Tony where he was removing the pageHeading from the components.

Ideally I would not group components, but that's just my personal opinion. I would not argue if you choose to keep it.

Reply:
As its not the primary purpose of this ticket I won't attempt to remove it now (would also need to update the original field_accessible_autocomplete_with_header.scala.html template at the same time). Maybe a separate UI ticket could be created for this work.